fly.toml

```diff
@@ -6,7 +6,7 @@
 app = 'nwc'
 primary_region = 'lax'
 swap_size_mb = 2048
-kill_timeout = 120
+kill_timeout = 300
```

Comment:
I noticed, this should not be exactly the same as the LDK timeout otherwise the hub could be killed while it's shutting down. Could we change it to 330?

Also can you do a PR inside getalby.com?